pycheribuild/projects/disk_image.py

```diff
@@ -818,7 +834,10 @@ def __process(self):
             # Add/symlink GDB (if requested).
             self.add_gdb()
             # finally create the disk image
-            self.make_disk_image()
+            if self.is_tarball:
```

Comment:
You could just override make_disk_image in the subclass?

Reply:
I had shied away from that because I thought we might want `minimal` tarballs, too, but yeah, that's probably a better plan.